test/test_attention.py

```diff
@@ -106,10 +106,6 @@ def test_benchmark_masked(
 
     assert_all_close(a=value.float(), b=expected, atol=1e-1)
 
-    for _ in range(10):
```

Comment:
do you remember why we have this ?

Reply:
Yeah it was for stability reason because when you have an issue in the load mask that test has some chance to catch it. It should not be required with mlir and we will get a faster CI